diff --git a/node_modules/next/dist/lib/get-network-host.js b/node_modules/next/dist/lib/get-network-host.js
index 091e292..42142a9 100644
--- a/node_modules/next/dist/lib/get-network-host.js
+++ b/node_modules/next/dist/lib/get-network-host.js
@@ -15,7 +15,12 @@ function _interop_require_default(obj) {
     };
 }
 function getNetworkHosts(family) {
-    const interfaces = _os.default.networkInterfaces();
+    let interfaces;
+    try {
+        interfaces = _os.default.networkInterfaces();
+    } catch (e) {
+        return family === 'IPv4' ? ['127.0.0.1'] : [];
+    }
     const hosts = [];
     Object.keys(interfaces).forEach((key)=>{
         var _interfaces_key;
